packages/application-shell/src/components/user-settings-menu/user-settings-menu.js

```diff
@@ -40,120 +40,184 @@ UserAvatar.propTypes = {
+                            <Text.Body isBold>
+                              {[this.props.firstName, this.props.lastName]
+                                .join(' ')
+                                .trim()}
+                            </Text.Body>
+                            <Text.Body truncate>{this.props.email}</Text.Body>
+                          </div>
+                        </Spacings.Inline>
                       </Spacings.Inset>
                     </div>
-                  </Link>
-                  <ToggleFeature flag={ORGANIZATIONS_LIST}>
-                    <Link to="/account/organizations" onClick={toggleMenu}>
+                    {!this.props.isAccountPath && (
+                      <React.Fragment>
+                        <Link to="/account/profile" onClick={toggleMenu}>
+                          <div className={styles.item}>
+                            <Spacings.Inset scale="s">
+                              <FormattedMessage {...messages.userProfile} />
+                            </Spacings.Inset>
+                          </div>
+                        </Link>
+                        <ToggleFeature flag={ORGANIZATIONS_LIST}>
+                          <Link
+                            to="/account/organizations"
+                            onClick={toggleMenu}
+                          >
+                            <div className={styles.item}>
+                              <Spacings.Inset scale="s">
+                                <FormattedMessage
+                                  {...messages.manageOrganizations}
+                                />
+                              </Spacings.Inset>
+                            </div>
+                          </Link>
+                        </ToggleFeature>
+                        <ToggleFeature flag={PROJECTS_LIST}>
+                          <Link to="/account/projects" onClick={toggleMenu}>
+                            <div
+                              className={classnames(
+                                styles.item,
+                                styles['item-divider-account-section']
+                              )}
+                            >
+                              <Spacings.Inset scale="s">
+                                <FormattedMessage
+                                  {...messages.manageProjects}
+                                />
+                              </Spacings.Inset>
+                            </div>
+                          </Link>
+                        </ToggleFeature>
+                      </React.Fragment>
+                    )}
+                    {
+                      // FIXME: added as urgent request for GDPR changes
+                      // Should be properly added on the CTP-1209 task
+                    }
+                    <a
+                      href={`https://commercetools.com/privacy`}
+                      target="_blank"
+                      onClick={toggleMenu}
```

Comment:
how come we need to toggleMenu here?

also can you add rel="noopener noreferrer"

Reply:
Some dark corner of my brain tells
me we had it and removed it. Can not remember the reason nor if it was good Maybe dig the blame a bit to see and if there is nothing change it 😇

Reply:
@tdeekens yeah since the links are "internal", there isn't a security concern here 😄.